frmSettings.Designer.cs

```diff
@@ -77,6 +81,8 @@ partial class frmSettings {
             this.btnCompletedFolder = new System.Windows.Forms.Button();
             this.chkCompletedFolder = new System.Windows.Forms.CheckBox();
             this.chkCompletedFolderRelative = new System.Windows.Forms.CheckBox();
+            this.frmSettingsToolTip = new System.Windows.Forms.ToolTip(this.components);
+            this.groupBox1 = new System.Windows.Forms.GroupBox();
```

Comment:
Not of great importance, but can this have a more explicit name like the other group boxes? Something like `grpThreadStatus` or what you think is best.

Reply:
Missed that one, thought I'd renamed it